cassiopeia/solver/MaxCutSolver.py

```diff
@@ -0,0 +1,96 @@
+"""
+"""
+import itertools
+import numpy as np
+import pandas as pd
+import networkx as nx
+from typing import Callable, Dict, List, Optional, Tuple
+
+from cassiopeia.solver import GreedySolver
+from cassiopeia.solver import graph_utils
```

Comment:
Let's be more specific here with this library's name - `greedy_graph_utilities.py`